bundle_result.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package bundle_result;
+
+// Indicates the bundle successfully won its auction and was dispatched to the current leader.
+message Dispatched {
+  // The bundle's placement within the auction upon completion (e.g. 1st or 2nd place).
+  uint32 auction_placement = 1;
+
+  // This bundle's bid.
+  uint64 originator_bid_lamports = 2;
+
+  // Window of slots this bundle was sent for.
+  repeated uint64 slot_window = 4;
+
+  // List of validators this bundle was dispatched to.
+  repeated string validator_identities = 5;
+}
+
+// Indicates the bundle was dropped and therefore not forwarded to any validator.
+message Dropped {
+  oneof reason {
+    FailedSimulation failed_simulation = 1;
+    InternalError internal_error = 2;
```

Comment:
leave `InternalError` to be a catchall, and move to the bottom?

Reply:
the placement within a `oneof` doesn't have any functional difference, this just a style thing?

Reply:
yea just a style thing. catchall is usually at the end (like when all the `switch` branches dont catch anything)